src/services/RSVPService.ts

```diff
@@ -22,6 +35,13 @@ export class RSVPService {
 
     return newRSVP;
   }
+
+  async getAllEventRSVPs(event: Event): Promise<RSVP[]> {
```

Comment:
maybe rename to getEventRSVPs? right now it sounds like getting all the events's RSVPs

Reply:
also jw here (probably remnant from before), why is the param an event instead of eventId?
no problem with it being event im just curious

Reply:
For your second comment, I think it's because I do event existence check first, so I figured I might as well just pass in the event object itself since I have it (I think this was my thought process when I first worked on attendance/rsvp stuff, but man it's been a while so idk)

Reply:
for the first question maybe idk I'm lowkey a bit lazy lol